public/app/features/dashboard-scene/settings/VersionsEditView.tsx

```diff
@@ -188,32 +189,30 @@ function VersionsEditorSettingsListView({ model }: SceneComponentProps<VersionsE
   const hasMore = model.versions.length >= model.limit;
   const isLastPage = model.versions.find((rev) => rev.version === 1);
 
-  if (viewMode === 'compare') {
-    return (
-      <Page navModel={navModel} pageNav={pageNav} layout={PageLayoutType.Standard}>
-        <VersionHistoryHeader
-          onClick={model.reset}
-          baseVersion={baseInfo?.version}
-          newVersion={newInfo?.version}
-          isNewLatest={isNewLatest}
-        />
-        {isLoading ? (
-          <VersionsHistorySpinner msg="Fetching changes&hellip;" />
-        ) : (
-          <VersionHistoryComparison
-            newInfo={newInfo!}
-            baseInfo={baseInfo!}
-            isNewLatest={isNewLatest!}
-            diffData={model.diffData}
-            onRestore={dashboard.onRestore}
-          />
-        )}
-      </Page>
+  const viewModeCompare = (
+    <VersionHistoryHeader
```

Comment:
I think this needs to be wrapped together, version comparison no longer works.

Reply:
Good point! I've fixed this and compare view now works